certora/specs/LibUbiquityPool.spec

```
@@ -0,0 +1,99 @@
+//////////////////////////////////////////////////
+// Formal Verification: LibUbiquityPool
```

RuboCop (1.86.0)

[fatal] unexpected token tCONSTANT
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)